index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/fddcccfd38.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Local TubeClone</title>
+</head>
+
+<body>
+    <!--header start-->
+    <header>
+        <nav class="flex-container">
+            <i class="fas fa-bars"></i>
+            <img class="logo" src="images/logo" alt="image of logo">
+            <form action="">
+                <input class="myInput" type="text" id="search" placeholder="Search"><i class="fas fa-search"></i>
```

Comment:
- [ ] Your search icon should be a `<button>`, just put your `i` inside the requested tag
![image](https://user-images.githubusercontent.com/53256139/91039764-c3420c80-e5d2-11ea-816d-6e6cd45de6c1.png)